superset/examples/multiformat_time_series.py

```diff
@@ -98,7 +98,7 @@ def load_multiformat_time_series(  # pylint: disable=too-many-locals
     for col in obj.columns:
         dttm_and_expr = dttm_and_expr_dict[col.column_name]
         col.python_date_format = dttm_and_expr[0]
-        col.dbatabase_expr = dttm_and_expr[1]
+        col.database_expr = dttm_and_expr[1]
```

Comment:
I think this value has been deprecated, but if you want to be safe, you can change it to `col.database_expression`